src/main/java/com/happystays/cqrs/core/events/eventstoremodel/EventModel.java

```diff
@@ -1,18 +1,25 @@
 package com.happystays.cqrs.core.events.eventstoremodel;
```

Comment:
1. Please rename EventModel.java as BaseEventStore.java
2. No need to package eventstore is required. keep inside events